### ISSUE TITLE
[5] Information is lost when text is resized to 200%

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#278]

**Comment by bjgordon** [comment]
@klepas to investigate. An issue in uikits current nav. Low priority. Only an issue for text-resizing. Most modern browsers have page zooming, which does not have this issue.

**Comment by bjgordon** [comment]
Reported to uikit https://github.com/AusDTO/gov-au-ui-kit/issues/473

**Comment by bjgordon** [comment]
Closing this issue as it's low priority - will leave it to be addressed upstream.
